client/actions/channels/channel_client.go

```diff
@@ -2,9 +2,9 @@ package channels
 
 import (
 	"errors"
+	"github.com/IBM/satcon-client-go/client/auth"
 	"net/http"
 
-	"github.com/IBM/satcon-client-go/client/auth"
 	"github.com/IBM/satcon-client-go/client/types"
```

Comment:
At the risk of being nitpicky, I'd really prefer that we keep the imports organized, with all the imports from a particular module being together. This happens in a number of places in the PR. This particular instance has an added problem in that it's the only change in the file. Please keep the changes (as in number of files changed at all as well as number of lines changed in each file) as minimal as feasible.